internal/brokers/manager.go

```diff
@@ -48,37 +48,27 @@ func NewManager(ctx context.Context, configuredBrokers []string, args ...Option)
 
 	// Set default options.
 	opts := options{
-		rootDir: "/",
+		brokerDir: "/etc/authd/broker.d",
```

Comment:
IIRC, we avoided that to not hardcode anymore on the caller side the exact path for the broker.d for integration tests. Do you think this will be still be compatible?

Reply:
Maybe the `rootDir` control should be on the general manager (maybe even listed on `consts.go` as well, as something like `defaultRootDir`)?

Reply:
Sure, we can use a const, but this one will never be changed, as it’s global and we will need to have test-by-test override.

Reply:
No, I think we just say there is no broker apart from the local one.

Reply:
Updated!